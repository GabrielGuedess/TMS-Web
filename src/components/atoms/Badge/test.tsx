import { render } from '@testing-library/react';

import { Badge } from '.';

describe('<Badge />', () => {
  it('should render the badge', () => {
    const { container } = render(<Badge title="Test" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
