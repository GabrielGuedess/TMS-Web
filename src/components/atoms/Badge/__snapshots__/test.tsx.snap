// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> should render the badge 1`] = `
<div
  class="relative w-min"
>
  <div
    class="absolute left-0 top-0 size-full rounded opacity-[16%] bg-primary-400"
  />
  <div
    class="flex px-[0.625rem] py-[0.313rem] text-primary-400"
  >
    <span
      class="z-10 text-center text-[0.813rem] font-semibold capitalize"
    >
      Test
    </span>
  </div>
</div>
`;
